templates/quiz/results.php

```diff
@@ -68,9 +68,9 @@
 						<h5 class="llms-content-block">
 							<?php
 							if ( $is_passing_score ) {
-								_e( apply_filters( 'lifterlms_quiz_passed','Passed' ), 'lifterlms' );
```

Comment:
I think that this wasn't working because way it was placed in filter.. This should work but I don't know how to test it.